pygotham/talks/models.py

```diff
@@ -90,7 +143,10 @@ class Talk(db.Model):
     )
 
     user_id = db.Column(db.Integer, db.ForeignKey('users.id'), nullable=False)
```

Comment:
When we move everything to the `Speaker` model, we should remove `user_id` from `Talk`.